app/queries/get_edition_for_content_store.rb

```diff
@@ -0,0 +1,18 @@
+module Queries
+  class GetEditionForContentStore
+    def self.call(content_id, locale, include_draft = false)
+      allowed_states = [:published, :unpublished]
+      allowed_states << :draft if include_draft
+
+      scope = Edition.with_document
+      scope = Unpublishing.join_editions(scope)
```

Comment:
probably nicer to have this as `.left_outer_join(:unpublishing)` I think we only have the `join_editions` because Edition didn't know about unpublishings